src/OpenTelemetry.Exporter.OpenTelemetryProtocol.PersistentStorage/OtlpExporterPersistentStorageExtensions.cs

```diff
@@ -0,0 +1,77 @@
+// <copyright file="OtlpExporterPersistentStorageExtensions.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+using OpenTelemetry.Exporter.OpenTelemetryProtocol.Implementation;
+using OpenTelemetry.Extensions.PersistentStorage.Abstractions;
+using OpenTelemetry.Internal;
+using OpenTelemetry.Trace;
+
+namespace OpenTelemetry.Exporter.OpenTelemetryProtocol.PersistentStorage;
+
+/// <summary>
+/// Extension methods to simplify registering of the OpenTelemetry Protocol (OTLP) exporter
+/// with persistent storage.
+/// </summary>
+public static class OtlpExporterPersistentStorageExtensions
```

Comment:
Maybe add "trace" into the name like "OtlpTraceExporterPersistentStorageExtensions". Could help with doc generation if we ever did it.

Reply:
Good call, I forgot that a new class/file will be created for logs and metrics later!